### PR TITLE
Changes the Keys Class

### PR DESCRIPTION
**Pull request by alallema**
### Breaking Changes

The `class` `Keys` wasn't used as a proper class and did not return objects from the `Keys` but arrays. This has been changed. Setters and Getters have been added to the class.

- `client.getKey` return a Keys
- `client.createKey` return a Keys
- `client.updateKey` return a Keys

### CODE REVIEW

File: src/Endpoints/Keys.php [hunk 1]
```diff
@@ -5,29 +5,133 @@
...
+            $attributes['expiresAt'],
+            $attributes['createdAt'],
+            $attributes['updatedAt'],
+        );
+    }
+
+    /**
+     * @return $this
+     */
+    protected function fill(array $attributes): self
```
Comment (brunoocasali) [inline src/Endpoints/Keys.php:52]: I'm not sure how it should be done, but it seems to me like a duplication, what about try to reuse this `fill` with the `newInstance`?

**Review by brunoocasali — COMMENTED** [review]
Well done! 🥇

File: src/Endpoints/Keys.php (hunk 1 above)
Comment (alallema) [reply src/Endpoints/Keys.php]: Yes it was I remove it

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by alallema** [comment]
bors merge

**Comment by brunoocasali** [comment]
@alallema is there any particular reason to merge this into the `bump` instead of main?

**Pull request merged**